Packs/ThreatGrid/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cisco Threat Grid
+- Fixed an issue where ***threat-grid-get-analysis-iocs*** was return an error in case there were no indicators, or it did not have a score
```

Comment:
```suggestion
- Fixed an issue where the ***threat-grid-get-analysis-iocs*** command returned an error in case there were no indicators, or indicators without a score.
```